src/components/Menudetails.js

```diff
@@ -0,0 +1,325 @@
+            <div className="mb-0 font-bold">
+              <span>$14</span>
+            </div>
+            <div className="ml-5 mb-0">
+              <RxCross2 />
+            </div>
+          </div>
+        </div>
+        <div className="flex mb-2 border-b-2">
+          <div className="w-2/3 pr-0">
+            <div className="mb-0">
+              <span>Pizza</span>
+            </div>
+          </div>
+          <div className="w-1/3 pl-0 text-right flex">
+            <div className="mb-0 font-bold">
+              <span>$10</span>
+            </div>
+            <div className="ml-5">
+              <RxCross2 />
+            </div>
+            {/* <p className="mb-0 font-bold"><span className="food-price">RS.{cartItemsList[val].itemPrice}</span><span onClick={() => this.removeCartItem(val)} className="remove-food-item"><FontAwesomeIcon icon="times" /></span></p> */}
+          </div>
+        </div>
+      </li>
+    );
+
+    // if (cartItemsList) {
+    //     return Object.keys(cartItemsList).map((val) => {
+    //         return (
+    //             <li className="food-item border-bottom pb-2 mb-3" key={val}>
+    //                 <div className="flex">
+    //                     <div className="col-8 pr-0">
+    //                         <p className="mb-0">{cartItemsList[val].itemTitle}</p>
+    //                     </div>
+    //                     <div className="col-4 pl-0 text-right">
+    //                         <p className="mb-0"><span className="food-price">RS.{cartItemsList[val].itemPrice}</span><span onClick={() => this.removeCartItem(val)} className="remove-food-item"><FontAwesomeIcon icon="times" /></span></p>
+    //                     </div>
+    //                 </div>
+    //             </li>
+    //         )
+    //     })
+    // }
+  };
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+  //   const classes = useStyles();
+
+  return (
+    <div>
+      <div className="container-fluid h-72 bg-cover bg-restaurant-cover">
+        <div>
+          <div className="container p-20 mx-0">
+            <div className="container">
+              <div className="flex">
+                <div className="items-center col-span-2">
+                  <img
+                    className="w-60 scale-75"
+                    alt="Food"
+                    src={require("../assets/images/rest_image.png")}
+                  />
+                </div>
+                <div className="pt-16">
+                  <h1 className="text-2xl text-white font-semibold">
+                    Olive Garden
+                  </h1>
+                  <p className="text-white">Food, Drinks</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="container-fluid bg-slate-200">
+        <div className="container mx-auto">
```

Comment:
The "Container-fluid" class is used in bootstrap.css. You can change that because It might be a problem while we do the integration of the modules.

Reply:
Made the changes